src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrCodeflower.cs

```diff
@@ -48,8 +49,17 @@ public abstract class VmrCodeFlower : IVmrCodeFlower
     private readonly ILocalGitRepoFactory _localGitRepoFactory;
     private readonly IVersionDetailsParser _versionDetailsParser;
     private readonly IFileSystem _fileSystem;
+    private readonly ICommentCollector _commentCollector;
     private readonly ILogger<VmrCodeFlower> _logger;
 
+    private static readonly string CannotFlowAdditionalFlowsInPrMsg = string.Concat(
+        "The source repository has received code changes from outside of this PR branch. This PR will stop ",
+        "receiving additional codeflow commits. In order to resume codeflows into this repository, this ",
+        "PR must be merged or closed. Once this is done, code flows into the repository will resume normally ",
+        "on the next subscription trigger. In order to force the subscription trigger, merge or close this PR " +
+        "and run the following command using the subscription-id found in this PR. \n",
+        "`darc trigger-subscription --id <subscriptionId>`.");
```

Comment:
think this has to be updated for the `force` parameter?

Reply:
but that is another option, no? if a person wants to trigger another subscription update in the same PR, they can use `force`